runtimes/aspnetcore-1.1.1/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This docker file installs the .NET core runtime.. The apps are expected to be
+# published before being added to the container to avoid having to resolve
+# properties furing the Docker build process.
+
+# This is an official app engine image.
+FROM gcr.io/google_appengine/debian8
+
+# Install .NET Core dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        libc6 \
+        libcurl3 \
+        libgcc1 \
+        libicu52 \
+        liblttng-ust0 \
+        libssl1.0.0 \
+        libstdc++6 \
+        libtinfo5 \
+        libunwind8 \
+        libuuid1 \
+        zlib1g \
+        ca-certificates \
+        curl \
+    && apt-get clean
+
+# Install the package.
+RUN mkdir -p /usr/share/dotnet && \
+    curl -sL https://storage.googleapis.com/aspnet-docker-deps/dotnet-debian-x64.1.1.1.tar.gz | tar -xz -C /usr/share/dotnet/ && \
```

Comment:
Would you mind moving this bucket to the project 'gcp-runtimes'? We have some shared monitoring setup for artifacts served from buckets in that project.